interface/drugs/drug_inventory.php

```diff
@@ -364,10 +364,10 @@ function facchanged() {
   <td>
    <input type='checkbox' name='form_show_empty' value='1'<?php if ($form_show_empty) {
         echo " checked";} ?> />
-   <?php echo xl('Show empty lots'); ?><br />
+   <?php echo xlt('Show empty lots'); ?><br />
    <input type='checkbox' name='form_show_inactive' value='1'<?php if ($form_show_inactive) {
         echo " checked";} ?> />
-   <?php echo xl('Show inactive'); ?>
+   <?php echo xlt('Show inactive'); ?>
```

Comment:
@sunsetsystems , fyi just made 3 very minor changes to your prior commit

Reply:
Thanks Brady!